protocol/chat.go

```diff
@@ -40,6 +40,23 @@ const (
 	FirstMessageTimestampNoMessage = 1
 )
 
+const (
+	MuteFor1MinDuration   = 60
+	MuteFor15MinsDuration = 900
+	MuteFor1HrsDuration   = 3600
+	MuteFor8HrsDuration   = 28800
+	MuteFor1WeekDuration  = 604800
+)
+
+const (
+	MuteFor15MinsType = 1
```

Comment:
You can also use a different type for this, to have a bit more type safety

Reply:
Hmmm, Do you suggest any specific type ?

Reply:
You create one, for enums is customary to have their own type, for example:

```
type ActivityCenterType int

const (
	ActivityCenterNotificationNoType ActivityCenterType = iota
	ActivityCenterNotificationTypeNewOneToOne
	ActivityCenterNotificationTypeNewPrivateGroupChat
	ActivityCenterNotificationTypeMention
	ActivityCenterNotificationTypeReply
	ActivityCenterNotificationTypeContactRequest
	ActivityCenterNotificationTypeCommunityInvitation
	ActivityCenterNotificationTypeCommunityRequest
	ActivityCenterNotificationTypeCommunityMembershipRequest
	ActivityCenterNotificationTypeCommunityKicked
	ActivityCenterNotificationTypeContactVerification
)

```

Note that the first type, is the "unknown" type, since if a value is not in the enum, it will default to that one in some cases.